spec/unit/util/windows/file_spec.rb

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env ruby
+require 'spec_helper'
+require 'puppet/util/windows/file'
```

Comment:
I think you want to move this file to `spec/integration/util/windows/file_spec.rb` and change this include to:

```ruby
require 'puppet/util/windows'
```

That should make sure it doesn't fail on non-Windows.